draft-ietf-rats-msg-wrap.md

```diff
@@ -436,6 +458,210 @@ The initial registrations for the registry are detailed in {{tab-ind-regs}}.
 | 3 | Attestation Results | {{&SELF}} |
 {: #tab-ind-regs title="CMW Indicators Registry Initial Contents"}
 
+## Media Types
+
+IANA is requested to add the following media types to the "Media Types" registry {{!IANA.media-types}}.
```

Comment:
Is there a media type for anything that a tagged CBOR (i.e., its a CBOR tags registered tag)? 
e.g.: "application/tagged+cbor"

Reply:
no